### ISSUE TITLE
UI accessibility with screen readers

### ISSUE THREAD

**Issue by RealAmethyst** [gopher64/gopher64#639]
The emulator is completely inaccessible for blind users who uses screen readers to navigate a computer. Depending on what UI framework this uses it should be possible to ad. I really want to use this emulator but I am unable to.
